src/styles/custom.css

```diff
@@ -691,6 +691,12 @@ kbd {
 	}
 }
 
+@media only screen and (max-width: 600px) {
+	.prev-next-step-grid-column {
+		grid-column: span 12 !important;
```

Comment:
Let's add the padding here `15px 0px`

Result: 
![image](https://user-images.githubusercontent.com/22376783/64151273-a0f63280-ce47-11e9-8fd6-41c58bc540f3.png)

previous:
![image](https://user-images.githubusercontent.com/22376783/64151292-abb0c780-ce47-11e9-9cc2-07a817f1bdc5.png)